.github/workflows/ci.yml

```
@@ -0,0 +1,62 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - trunk
+  pull_request:
+
+jobs:
+  php-cs-check:
+    name: 'Check PHP CS == 7.4'
```

The name suggests this checks whether the PHPCS version is `7.4` but I think the intention was to:
```suggestion
    name: 'PHP 7.4 compatibility (via PHPCS)'
```

---

`CS` was for `Coding Standards`, but your option leaves no room for misunderstandings. Also, this is a whoosh moment for me. I did not realize it could mean Code Sniffer, too. Fixed.